src/debian/sudoers/ceph.j2

```diff
@@ -0,0 +1,13 @@
+# This file overwrite a sudoers file created by ceph.
+# The original file contains two rules :
+#
+# ceph ALL=NOPASSWD: /usr/sbin/smartctl -a --json=o /dev/*
+# ceph ALL=NOPASSWD: /usr/sbin/nvme * smart-log-add --json /dev/*
+#
+# The first rule is modified to target the right disk instead of a wildcard
+# The second one is remove cause the needed packages are not installed
+#
+# The replace directive add an escape backslash before the colon character
+# for the file to be properly parsed.
+
+ceph ALL=NOPASSWD: /usr/sbin/smartctl -a --json=o {{ ceph_osd_disk if ceph_osd_disk is defined else none | replace(':', '\\:') }}
```

Comment:
There is a warning using sudo with this rule:
```
ceph ALL=NOPASSWD: /usr/sbin/smartctl -a --json=o /dev/disk/by-path/pci-0000:c3:00.0-scsi-0:2:1:0
``` 

Futhermore ceph uses /dev/sdX as path.